README.md

```diff
@@ -1,74 +1,95 @@
-# FFC Pay Report Generator
+# Payment Hub Report Generator
 
-Template to support rapid delivery of microservices for FFC Platform. It contains the configuration needed to deploy a simple Hapi Node server to the Azure Kubernetes Platform.
+Microservice to periodically generate reports from Payment Hub event store.
+
+This service is part of the [Payment Hub](https://github.com/DEFRA/ffc-pay-core).
 
 ## Prerequisites
 
-- Docker
-- Docker Compose
+- [Docker](https://www.docker.com/)
+- Either:
+  - [Docker Compose](https://docs.docker.com/compose/install/linux/#install-the-plugin-manually)
+  - [Docker-Compose (standalone)](https://docs.docker.com/compose/install/other/)
 
 Optional:
-- Kubernetes
-- Helm
-
-## Running the application
-
-The application is designed to run in containerised environments, using Docker Compose in development and Kubernetes in production.
+- [Kubernetes](https://kubernetes.io/)
+- [Helm](https://helm.sh/)
 
-- A Helm chart is provided for production deployments to Kubernetes.
+## Setup
 
-### Build container image
+### Configuration
 
-Container images are built using Docker Compose, with the same images used to run the service with either Docker Compose or Kubernetes.
+These configuration values should be set in the [docker-compose.yaml](docker-compose.yaml) file or Helm [values file](helm/ffc-pay-report-generator/values.yaml) if running Kubernetes.
 
-When using the Docker Compose files in development the local `app` folder will
-be mounted on top of the `app` folder within the Docker container, hiding the CSS files that were generated during the Docker build.  For the site to render correctly locally `npm run build` must be run on the host system.
+| Name | Description |
+| ---| --- |
+| `APPINSIGHTS_CLOUDROLE` | Azure App Insights cloud role |
+| `APPINSIGHTS_CONNECTIONSTRING` | Azure App Insights connection string |
 
+#### Docker
 
-By default, the start script will build (or rebuild) images so there will
-rarely be a need to build images manually. However, this can be achieved
-through the Docker Compose
-[build](https://docs.docker.com/compose/reference/build/) command:
+Docker Compose can be used to build the container image.
 
 ```
-# Build container images
 docker-compose build
 ```
 
-### Start
+The service will file watch application and test files so no need to rebuild the container unless a change to an npm package is made.
```

Comment:
The description of 'file watch application' doesn't make much sense to me here, but that could just be me being tired

Reply:
Reworded, any better?